# TrajClient params
T_horizon: 50
X_des: [8.000000,0.300000,0.000000,0.000000,0.000000,0.000000]
timestep: 0.02

# Rampup params
kp_ramp: 0.45
kd_ramp: 0.05
ki_ramp: 0.1
accel_ramp: 3
target_vel_ramp: 3
timeout_ramp: 2.5

# iLQR params
timeout_ilqr_mpc: 5
stop_goal_threshold: 0.3

ilqr_tolFun: 0.0000001
ilqr_tolConstraint: 0.0000001
ilqr_tolGrad: 0.00001
ilqr_max_iter: 20
ilqr_regType: 1
ilqr_debug_level: 2

init_control_seq: [0.139439,-0.105075,0.280323,0.011749,0.416098,-0.082468,0.483120,0.054050,0.522686,0.029919,0.577766,-0.114025,0.600918,-0.016742,0.598326,-0.057170,0.560930,0.019255,0.623788,0.040867,0.706408,0.049813,0.757708,0.163735,0.822205,0.177798,0.916419,0.205154,0.969974,0.257718,1.028822,0.220097,1.122343,0.267409,1.180215,0.273489,1.242806,0.234608,1.300371,0.230437,1.360076,0.229049,1.420983,0.217776,1.478654,0.221562,1.533587,0.222081,1.585164,0.213202,1.636787,0.208404,1.688093,0.191469,1.739682,0.162888,1.790205,0.124738,1.839207,0.081811,1.885776,0.039103,1.923508,-0.058946,1.950566,-0.149644,1.970970,-0.220538,1.983464,-0.273418,1.988660,-0.314000,1.986933,-0.341757,1.980237,-0.357679,1.971503,-0.361582,1.964121,-0.353117,1.960646,-0.332663,1.961812,-0.302756,1.966272,-0.266119,1.970891,-0.228886,1.974633,-0.190749,1.974605,-0.161061,1.967512,-0.128432,1.961286,-0.110812,1.968319,-0.087347,1.892385,-0.016616]

Opt_car_param: {g: 9.81, m: 2.35, L: 0.257, b: 0.14328,
                c_x: 116, c_a: 197, Iz: 0.025, mu: 1.31, mu_s: 0.55,
                limThr: [0, 2.0], limSteer: [-0.68, 0.76]}
Opt_cost: {cu: [0.01, 0.01], cdu: [1.0, 1.5],
           cf: [10, 10, 5, 10, 0.1, 0.1],
           pf: [0.01, 0.01, 0.1, 0.1, 0.1, 0.1],
           cx: [0.05, 0.05, 0.04], cdx: [0.001, 0.005, 0.002],
           px: [0.01, 0.01, 0.1],
           cdrift: -0.001,
           k_pos: 0.5, k_vel: 0, d_thres: 0.3}

######
# TrajServer params
old_msg_discard_thres: 0.5

# Obstacle detection params
scan_clip_angle: 0.35 #radians
naive_obstacle_dist_thres: 1.5
naive_obstacle_percent_thres: 0.2
